src/Libraries/Microsoft.Extensions.Resilience/Resilience/Internal/ResilienceTagNames.cs

```diff
@@ -5,13 +5,13 @@ namespace Microsoft.Extensions.Resilience.Internal;
 
 internal static class ResilienceTagNames
 {
-    public const string FailureSource = "failure-source";
+    public const string FailureSource = "failure.source";
 
-    public const string FailureReason = "failure-reason";
+    public const string FailureReason = "failure.reason";
 
-    public const string FailureSummary = "failure-summary";
+    public const string FailureSummary = "failure.summary";
 
-    public const string DependencyName = "dep-name";
+    public const string DependencyName = "dependency.name";
 
-    public const string RequestName = "req-name";
+    public const string RequestName = "request.name";
```

Comment:
Are these tags reused from semantic conventions or specific to resilience? If they are specific, then maybe they should be prefixed to avoid clashing with other tag names, e.g. `resilience.failure.source`.

Reply:
Done, thanks!
@martintmk FYI (again 😄)